### DISCUSSION TITLE
Will ULID be added to Prisma as ID generation method?

### DISCUSSION THREAD

**Discussion by keinsell** [prisma/prisma#19524] [Q&A] [answered]
https://github.com/ulid/spec

**Comment by nurul3101** [comment, marked as answer]
We have a feature request for supporting ULID here:
- https://github.com/prisma/prisma/issues/13679

Please add a 👍 to the feature request so that we can prioritise supporting it.

**Reply by keinsell** [reply]
That resolves my question, I've upvoted feature request.
